DEVELOPING_TUTORIALS.md

```diff
@@ -639,48 +562,40 @@ Note that your user history on the live website (https://proto.school) is differ
+That's it! Next time you run ProtoSchool the tutorial should be renamed and users will still have access to the status of their lesson progress. If a user tries to access the old URL for your tutorial or one of its lessons, they'll be redirected to the tutorial landing page at its new URL.
 
-```js
-"0005": {
-  "url": "new-short-tutorial-title",
-  "project": "IPFS",
-  "title": "New name of this tutorial",
-  "description": "Description of tutorial",
-  // etc.
-```
+### My files (.md, -exercise.md) content are not showing up in the lesson page
 
-Then in the `src/main.js` file, find the routes for your tutorial by its ID (which will remain unchanged) and update the paths to reflect the new `url` value in `static/tutorials.json`:
+The file might not be loading correctly. You can use the `DEBUG=true` option to get warnings in the console. To enable it run:
 
-```js
-// Tutorial 0005
-{ path: '/new-short-tutorial-title', component: Landing, props: { tutorialId: '0005' } },
-{ path: '/new-short-tutorial-title/resources', component: ResourcesLesson, props: { tutorialId: '0005' } },
-{ path: '/new-short-tutorial-title/01', component: T0002L01, props: { tutorialId: '0005', lessonId: '01' } },
-{ path: '/new-short-tutorial-title/02', component: T0002L02, props: { tutorialId: '0005', lessonId: '02' } },
-{ path: '/new-short-tutorial-title/03', component: T0002L03, props: { tutorialId: '0005', lessonId: '03' } },
+```sh
+DEBUG=true npm start
 ```
 
-In case someone uses an old link shared with them before the tutorial name and URL were changed, you'll also need to add two redirect objects as follows.
+Check the browser dev tools and check for warning logs.
 
-```js
-// Tutorial 0005
-{ path: '/old-short-tutorial-name', redirect: '/new-short-tutorial-name' },
-{ path: '/old-short-tutorial-name/*', redirect: '/new-short-tutorial-name' },
-```
+## Detailed Docs
 
-Finally, add a new object to the `MIGRATIONS` array, with the tutorial ID (unchaged) and the past URL:
+### Lesson file
 
-```js
-const MIGRATIONS = [
-  // { tutorialId: '0003', pastUrl: 'blog' }
-  { tutorialId: '0005', pastUrl: 'old-short-tutorial-name' }
-]
-```
+Lesson files should be put in the folder `src/tutorials/xxxx-tutorials-short-title` with the appropriate number (e.g., `01.js`).
 
-That's it! Next time you run ProtoSchool the tutorial should be renamed and users will still have access to the status of their lesson progress. If a user tries to access the old URL for your tutorial or one of its lessons, they'll be redirected to the tutorial landing page at its new URL.
+**Exportable properties**:
+
+- `validate(result, ipfs)`: Function - mandatory
+- `solution`: String - mandatory
+- `code`: String - optional
+- `modules`: Object - optional
+- `options`: Object - optional
+    - `type`: String - one of `standard` (default), `file-upload` or `multiple-choice`.
```

Comment:
I don't think this matches what you did when you updated the existing lessons. Didn't you use code and text? Or are we using different labels in JavaScript files than in Markdown files? Can we make them consistent if that's the case?